api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+	<changeSet id="43889130-55c2-49e9-b72b-a9eae9a31da7" author="Partners In Health">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="idgen_identifier_source"/></not>
+		</preConditions>
+		<createTable tableName="idgen_identifier_source">
+			<column name="id" type="int(11)" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" unique="true"/>
+			</column>
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="name" type="varchar(255)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="description" type="varchar(1000)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="identifier_type" type="int(11)" defaultValue="0">
+				<constraints nullable="false" unique="true" foreignKeyName="identifier_type for idgen_identifier_source" references="patient_identifier_type(patient_identifier_type_id)"/>
+			</column>
+			<column name="creator" type="int(11)" defaultValue="0">
+				<constraints nullable="false" unique="true" foreignKeyName="creator for idgen_identifier_source" references="users(user_id)"/>
+			</column>
+			<column name="date_created" type="datetime">
+				<constraints nullable="false"/>
+			</column>
+			<column name="changed_by" type="int(11)" defaultValue="NULL">
+				<constraints nullable="true" unique="true" foreignKeyName="changed_by for idgen_identifier_source" references="users(user_id)"/>
+			</column>
+			<column name="date_changed" type="datetime" defaultValue="NULL">
```

Comment:
Let us just drop the default attribute for this.

Reply:
oaky.